src/Billing/EDI270.php

```diff
@@ -856,9 +859,9 @@ public static function requestEligibility($partner = '', $x12_270 = '')
         $formBody = $response->body();
```

Comment:
hi @sjpadgett , I'm getting `Error:Http Error: Bad Request : 400` on testing. How do I see the validation errors that are in `$formBody`?

Reply:
I'm getting same thing today. Need to check with user if he to if not working. 
Uncheck
![image](https://github.com/user-attachments/assets/50738c6b-d535-43c6-b308-006048343d26)

![image](https://github.com/user-attachments/assets/3353b8bb-b143-405b-ba1c-e680ffce8398)